detox/test/e2e/c-stressful-use.js

```diff
@@ -0,0 +1,41 @@
+const MULTI_TEST_COUNT = 35
+
+describe.only('Stressful Use', function() {
+  describe('Stress on React Bridge', function() {
```

Comment:
we don't need so many `describe` blocks, have just one `describe` on top and put all the tests as `it` inside this describe, we don't need a describe per test